components/session.py

```diff
@@ -4,19 +4,20 @@
 from discord.ext import tasks
 from typing import Optional
 
-from components.bot import RecruitBot
 from components.views import SessionView
 
 
 class Session:
-    bot: RecruitBot
+    # bot: RecruitBot
     user: discord.User
     interval: int = 45
     strikes: int = 0
     channel: Optional[discord.TextChannel] = None
 
-    def __init__(self, bot: RecruitBot, user: discord.User, channel_id: int, interval):
-        self.bot = bot
+    def __init__(self, bot, user: discord.User, channel_id: int, interval):
+        from components.bot import RecruitBot

```

Comment:
Why import here instead of at the top?

Reply:
It wouldn't compile otherwise due to circular imports!